test/OptionSettlementEngine.t.sol

```diff
@@ -775,13 +788,13 @@ contract OptionSettlementTest is Test, NFTreceiver {
         uint256 cTokenId2 = engine.write(oTokenId, 3);
 
         // Check encoding the first claim
-        (uint160 decodedOptionId,) = engine.decodeTokenId(cTokenId1);
+        (uint160 decodedOptionId,) = decodeTokenId(cTokenId1);
```

Comment:
Not related to your change, but this local var should be `decodedOptionKey`

Reply:
Will hit in the test cleanup PR.